lintreview/tasks.py

```diff
@@ -59,14 +60,20 @@ def process_pull_request(user, repo_name, number, lintrc):
         log.info('Completed lint processing for %s/%s/%s' % (
             user, repo_name, number))
 
-    except BaseException as e:
```

Comment:
The [docs for BaseException](https://docs.python.org/2.7/library/exceptions.html#exceptions.BaseException) say:

> The base class for all built-in exceptions. It is not meant to be directly inherited by user-defined classes (for that, use Exception).